### PR TITLE
PUB-60 : Deprecate v1,v2 style subscribers

### PR DESCRIPTION
**Pull request by rohit-gohri**
Closes PUB-60

### CODE REVIEW

File: src/subscriber/subscriberV2.ts [hunk 1]
```diff
@@ -62,6 +64,19 @@ export default class SubscriberV2 extends Subscriber {
             },
           };
 
+          constructor(...args: any[]) {
```
Comment (rohit-gohri) [inline src/subscriber/subscriberV2.ts:67]: Codacy has a fix for the issue: [Unexpected any. Specify a different type.](https://app.codacy.com/gh/deliveryhero/hfc-pubsub/pullRequest?prid=8095345)
```suggestion
          constructor(...args: unknown[]) {
```

File: src/subscriber/subscriberV2.ts [hunk 2]
```diff
@@ -1,4 +1,6 @@
+import util from 'util';
 import { SubscriberOptions as GoogleCloudSubscriberOptions } from '@google-cloud/pubsub/build/src/subscriber';
+import { Logger } from '../service/logger';
```
Comment (rohit-gohri) [inline src/subscriber/subscriberV2.ts:3]: Codacy has a fix for the issue: [Replace `{·Logger·}·from·'../service/logger` with `util·from·'util`](https://app.codacy.com/gh/deliveryhero/hfc-pubsub/pullRequest?prid=8095345)
```suggestion
import util from 'util';
```

File: src/subscriber/subscriberV2.ts [hunk 3]
```diff
@@ -94,6 +109,18 @@ export default class SubscriberV2 extends Subscriber {
         };
 
         return class extends subscriberClass {
+          constructor(...args: any[]) {
```
Comment (rohit-gohri) [inline src/subscriber/subscriberV2.ts:112]: Codacy has a fix for the issue: [Unexpected any. Specify a different type.](https://app.codacy.com/gh/deliveryhero/hfc-pubsub/pullRequest?prid=8095345)
```suggestion
          constructor(...args: unknown[]) {
```

File: src/client/googlePubSub.ts [hunk 4]
```diff
@@ -1,15 +1,15 @@
...
 import {
   PubSub as GooglePubSub,
   Message as GoogleCloudMessage,
   Subscription as GoogleCloudSubscription,
   Topic as GoogleCloudTopic,
 } from '@google-cloud/pubsub';
 import { Resource } from '@google-cloud/resource';
-
 import { CredentialBody } from 'google-auth-library';
+import chalk from 'chalk';
```
Comment (rohit-gohri) [inline src/client/googlePubSub.ts:10]: Codacy has a fix for the issue: [Replace `chalk·from·'chalk` with `util·from·'util`](https://app.codacy.com/gh/deliveryhero/hfc-pubsub/pullRequest?prid=8095345)
```suggestion
import util from 'util';
```

File: src/subscriber/subscriberV2.ts (hunk 2 above)
Comment (rohit-gohri) [inline src/subscriber/subscriberV2.ts:3]: Codacy has a fix for the issue: [Replace `{·Logger·}·from·'../service/logger` with `util·from·'util`](https://app.codacy.com/gh/deliveryhero/hfc-pubsub/pullRequest?prid=8095345)
```suggestion
import util from 'util';
```

**Review by ishan123456789 — APPROVED** [review]
LGTM

**Pull request merged**